wallet-new/src/Cardano/Wallet/Kernel/DB/Spec.hs

```diff
@@ -252,6 +328,8 @@ class IsCheckpoint c where
     cpBlockMeta   :: Lens' c LocalBlockMeta
     cpForeign     :: Lens' c Pending
     cpContext     :: Lens' c (Maybe BlockContext)
+    delta         :: c -> c -> DeltaCheckpoint
+    step          :: c -> DeltaCheckpoint -> c
```

Comment:
Let's be consistent here with the rest also. So, `cpFindDelta` and `cpApplyDelta`.